src/account.ts

```diff
@@ -284,12 +284,38 @@ export class AccountApi {
      * message for recipient
      *
      * @param {number} amount
-     * @param {string} fromAccount      Name or id of account
-     * @param {string} toAccount        Name or id of account
+     * @param {string} fromAccount      Id of account
```

Comment:
what is diiference between accountName and accountId?

Reply:
account name is md5 hash of email starts with 'u', account id is in format '1.2.X'